packages/perseus/src/widgets/expression/score-expression.ts

```diff
@@ -165,9 +165,6 @@ function scoreExpression(
     // We matched a graded answer form, so we can now tell the user
     // whether their input was correct or incorrect, and hand out
     // points accordingly
-    // TODO(eater): Seems silly to translate result to this
```

Comment:
idk, I think we're past the point of doing anything about this since this is from the Eater era.